apps/web/src/app/admin-console/organizations/members/members.component.html

```diff
@@ -1,4 +1,15 @@
 @let organization = this.organization();
+@let bulkActions = bulkFlags$ | async;
+
+@if (!organization || !dataSource) {
+  <i
+    class="bwi bwi-spinner bwi-spin tw-text-muted"
+    title="{{ 'loading' | i18n }}"
+    aria-hidden="true"
+  ></i>
+  <span class="tw-sr-only">{{ "loading" | i18n }}</span>
+}
+
 @if (organization) {
```

Comment:
Does this also need to check for `dataSource` (the inverse of the loading condition)?

i.e. 
```suggestion
@if (organization && dataSource) {
```

(may need to be converted into a Signal)